sources/web/datalab/polymer/test/file-browser-test.ts

```diff
@@ -44,6 +44,20 @@ describe('<file-browser>', () => {
     new MockFile('file3'),
   ];
 
+  const fileAddButtonIds = [
+    'addNotebookButton',
+    'addFileButton',
+    'addFolderButton',
+  ];
+  const fileOpsButtonIds = [
```

Comment:
Consider splitting this into two lists such as 'singleEnabledButtons' and 'multipleEnabledButtons', and writing helper methods assertAllEnabled(buttonsList, message) and assertNoneEnabled(buttonList, message).

Reply:
Done.